clients/da-vinci-client/src/main/java/com/linkedin/davinci/consumer/VeniceChangelogConsumerImpl.java

```diff
@@ -377,12 +385,15 @@ public CompletableFuture<Void> subscribeAll() {
 
   @Override
   public CompletableFuture<Void> seekToTimestamps(Map<Integer, Long> timestamps) {
+    return internalSeekToTimestamps(timestamps, ChangeCaptureView.CHANGE_CAPTURE_TOPIC_SUFFIX);
```

Comment:
As we fixed the suffix type here, So seekToTimestamps and seekToTail would only meaningful to ChangeEventConsumer not afterImageConsumer?

Reply:
That is not the case.  We override this method in the afterimage consumer so as to not provide the suffix.  It's meaningful to both.